tests/test_trfdecode/test_decode.py

```diff
@@ -44,11 +42,6 @@
     os.path.dirname(__file__), "../data/trf")
 
 
-pytest.skip(
-    "Skipping decode_trf tests, the library can't be made public.",
-    allow_module_level=True)
-
```

Comment:
@Centrus007 :) Ahh, it be nice to be running these tests once more :)

Reply:
Woot!